specs/LANDRegistry.spec.js

```diff
@@ -0,0 +1,37 @@
+import { expect } from 'chai'
+import { LANDRegistry } from '../src/contracts/LANDRegistry'
+
+describe('LANDRegistry', function() {
```

Comment:
I was going to ask for a test before even seeing this file.
🥇

Reply:
😇